README.md

```diff
@@ -77,14 +77,28 @@ Place it into project library path in your IDE.
 ## Usage
 
 1. [Class Tarantool](#class-tarantool)
-1. [Predefined Constants](#predefined-constants)
+2. [Predefined Constants](#predefined-constants)
+3. [Manipulation connection](#manipulation-connection)
+  * [connect](#connect)
+  * [disconnect, close](#disconnect-close)
+  * [authenticate](#authenticate)
+  * [flushSchema, flush_schema](#flushschema-flush_schema)
+  * [ping](#ping)
+4. [Database queries](#database-queries)
+  * [select](#select)
+  * [insert, replace](#insert-replace)
+  * [call](#call)
+  * [eval, evaluate](#eval-evaluate)
+  * [delete](#delete)
+  * [update](#update)
+  * [upsert](#upsert)
 
 ### Class Tarantool
 
 _**Description**_: Creates a Tarantool client
 
 ``` php
-tarantool_object = new Tarantool([host = 'localhost'[, port = 3301]])
+$tarantool_object = new Tarantool([$host = 'localhost'[, $port = 3301]])
```

Comment:
I'm not sure it's a good idea - this file is library overview. In my opinion, it should contain library usage examples.
Maybe, this project needs API where you can find all methods and discover detailed information on them.
But. usually, first page contains real-usage examples that are syntax correct and can be copy-pasted.